docs/src/examples/QPagination/Icons.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="q-pa-lg flex flex-center">
+    <q-pagination
+      v-model="current"
+      :max="5"
+      :direction-links="true"
+      :boundary-links="true"
+      :icon-first="laAngleDoubleLeftSolid"
+      :icon-last="laAngleDoubleRightSolid"
+      :icon-prev="lasAngleLeft"
+      :icon-next="lasAngleRight"
+    >
+    </q-pagination>
+  </div>
+</template>
+
+<script>
+import {
+  laAngleDoubleLeftSolid, laAngleDoubleRightSolid, lasAngleLeft, lasAngleRight
+} from '@quasar/extras/line-awesome'
+
```

Comment:
Only comment would be to avoid using code that would break on UMD, otherwise codepen will stop working.
Try to use only Material Icons (webfont, not svg). If it's not possible to do something nice, then we drop the example.

Reply:
Got it, will make appropriate changes